packages/truffle-debugger/lib/data/actions/index.js

```diff
@@ -18,10 +18,9 @@ export function declare(node) {
 }
 
 export const ASSIGN = "ASSIGN";
-export function assign(context, assignments) {
```

Comment:
`context` wasn't being used anymore, was it?

Reply:
Yup.  Not sure it ever was while I was here...